src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ConditionalOrder.java

```diff
@@ -23,4 +36,77 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateConditionalOrderSubmitted;
 
+    @CCD(
+        label = "Link to respondent answers",
+        access = {DefaultAccess.class}
+    )
+    private Document respondentAnswersLink;
+
+    @CCD(
+        label = "Does the petitioner want to continue with the divorce and apply for a decree nisi?",
+        access = {DefaultAccess.class}
```

Comment:
you can remove default access from all fields in this class and just put it on the unwrapped field in CaseData